javascript/ql/src/semmle/javascript/security/dataflow/UrlConcatenation.qll

```diff
@@ -6,6 +6,27 @@
 
 import javascript
 
+/**
+ * Holds if the given data flow node is incomplete or is a string concatenation
```

Comment:
I think it would be clearer with a docstring along the lines of:

> Holds if the given data flow node may refer to a string for which we have incomplete information.

And a name of:

> mayBeIncompleteString

Reply:
The qldoc lgtm, though I don't really agree with the name. It's currently consistent with the other predicates in this file with names of form `hasXSubstring`.